en/README.md

```diff
@@ -88,8 +91,8 @@ The [Contributing Guide](contributing/contributing.md) explains the contribution
 
 ## License
 
-The message definition XML files and the generated [C-language version of MAVLink](#prebuilt_libraries) (a header-only library) are made available under the MIT-licence. MAVLink can therefore be *used* without limits in any closed-source application without publishing the source code of the closed-source application. See the [COPYING](https://github.com/mavlink/mavlink/blob/master/COPYING) file for more information.
+The message definition XML files and the generated [C-language version of MAVLink](#prebuilt_libraries) (a header-only library) are made available under the MIT-licence. MAVLink can therefore be *used* in any closed-source application without publishing the source code of the closed-source application. See the [COPYING](https://github.com/mavlink/mavlink/blob/master/COPYING) file for more information.
 
-The [MAVLink generator toolchain](https://github.com/mavlink/mavlink/) is licensed under the terms of the Lesser General Public License (version 3) of the Free Software Foundation (LGPLv3). Changes to the protocol/generator toolchain must therefore be contributed back to the project.
+The [MAVLink generator toolchain](https://github.com/mavlink/mavlink/) is licensed under the terms of the Lesser General Public License (version 3) of the Free Software Foundation (LGPLv3).
```

Comment:
Is "Changes to the protocol/generator toolchain must therefore be contributed back to the project." not true, or simply "not something you think needs to be said?

Reply:
It’s absolutely untrue.

Reply:
OK. In general licensing is a bitch. I'd prefer to change as you have suggested and people can then inform themselves properly of the meaning of the license.

Reply:
@hamishwillee The statement above was indeed correct, but referring to pymavlink (as it was in this repo). We also don't want to go into the specifics of when GPL requires you to contribute code and rather want to express general intent. E.g. if you do not redistribute your changed generator ever you technically don't need to contribute them back, but no general counsel will interpret that as "you don't have to contribute back".

I'm ok to not provide any any interpretation of the license itself as that's on the safe side.